sycl/test-e2e/Basic/vector/byte.cpp

```diff
@@ -92,212 +104,249 @@ int main() {
-    vop = vop1 & vop2;
-    vop = vop1 | vop2;
-    vop = vop1 ^ vop2;
-
-    // logical binary op for 2 swizzle
-    auto swand = swlo & swhi;
-    auto swor = swlo | swhi;
-    auto swxor = swlo ^ swhi;
-
-    // logical binary op for 1 vec
-    vop = vop1 & std::byte{3};
-    vop = vop1 | std::byte{3};
-    vop = vop1 ^ std::byte{3};
-    vop = std::byte{3} & vop1;
-    vop = std::byte{3} | vop1;
-    vop = std::byte{3} ^ vop1;
-
-    // logical binary op for 1 swizzle
-    auto swand2 = swlo & std::byte{3};
-    auto swor2 = swlo | std::byte{3};
-    auto swxor2 = swlo ^ std::byte{3};
-
-    auto swand3 = std::byte{3} & swlo;
-    auto swor3 = std::byte{3} | swlo;
-    auto swxor3 = std::byte{3} ^ swlo;
-
-    // bit binary op for 2 vec
-    vop = vop1 && vop2;
-    vop = vop1 || vop2;
-    vop = vop1 >> vop2;
-    vop = vop1 << vop2;
-
-    vop = vop1 >> std::byte{3};
-    vop = vop1 << std::byte{3};
-    vop = std::byte{3} >> vop1;
-    vop = std::byte{3} << vop1;
-
-    // bit binary op for 2 swizzle
-    swlo >> swhi;
-    swlo << swhi;
-    swlo >> std::byte{3};
-    swlo << std::byte{3};
-    auto right = std::byte{3} >> swhi;
-    auto left = std::byte{3} << swhi;
-
-    // condition op for 2 vec
-    auto vres = vop1 == vop2;
-    vres = vop1 != vop2;
-    vres = vop1 > vop2;
-    vres = vop1 < vop2;
-    vres = vop1 >= vop2;
-    vres = vop1 <= vop2;
-
-    vres = vop1 == std::byte{3};
-    vres = vop1 != std::byte{3};
-    vres = vop1 > std::byte{3};
-    vres = vop1 < std::byte{3};
-    vres = vop1 >= std::byte{3};
-    vres = vop1 <= std::byte{3};
-
-    vres = std::byte{3} == vop1;
-    vres = std::byte{3} != vop1;
-    vres = std::byte{3} > vop1;
-    vres = std::byte{3} < vop1;
-    vres = std::byte{3} >= vop1;
-    vres = std::byte{3} <= vop1;
-
-    // condition op for 2 swizzle
-    auto swres = swhi == swlo;
-    auto swres1 = swhi != swlo;
-    auto swres2 = swhi > swlo;
-    auto swres3 = swhi < swlo;
-    auto swres4 = swhi >= swlo;
-    auto swres5 = swhi <= swlo;
-    auto swres6 = swhi == std::byte{3};
-    auto swres7 = swhi != std::byte{3};
-    auto swres8 = swhi > std::byte{3};
-    auto swres9 = swhi < std::byte{3};
-    auto swres10 = swhi >= std::byte{3};
-    auto swres11 = swhi <= std::byte{3};
-
-    sycl::vec<std::byte, 3> voptest{std::byte{4}, std::byte{9}, std::byte{25}};
-    auto bitv1 = ~vop3;
-    auto bitv2 = !vop3;
-    auto bitw = ~swhi;
+    sycl::vec<std::byte, 3> VecByte3A{std::byte{4}, std::byte{9},
+                                      std::byte{25}};
+    sycl::vec<std::byte, 3> VecByte3B{std::byte{2}, std::byte{3}, std::byte{5}};
+    sycl::vec<std::byte, 4> VecByte4A{std::byte{5}, std::byte{6}, std::byte{2},
+                                      std::byte{3}};
+
+    // Test bitwise operations on vec<std::byte> and swizzles.
+    // Adding asserts on vec<> operations, and not swizzle operations,
+    // should suffice as swizzles just delegates the operation to vec<>
+    // class.
```

Comment:
Is this new or simply stating what was done before? It doesn't sound like a right thing to do to me.

Reply:
It's new - I've added new asserts only for `sycl::vec<>` operations, not swizzles.